### PR TITLE
specification for donut and sparkline chart component

### PR DESCRIPTION
**Pull request by kamleshverma1**

### CODE REVIEW

**Comment by kamleshverma1** [comment]
Against #84

**Comment by kamleshverma1** [comment]
@brainfunked @gnehapk Please review it..

**Comment by brainfunked** [comment]
@cloudbehl please review.

**Review by cloudbehl — COMMENTED** [review]
LGTM

**Review by cloudbehl — APPROVED** [review]

**Review by brainfunked — APPROVED** [review]
Looks good enough to me, but I'm not a UI developer.

**Comment by kamleshverma1** [comment]
@gnehapk Please review it.

File: specs/donut_and_sparkline_chart_component.adoc [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+
+=== Data model impact:
+
+See 'Impacted Modules' section where all the details are populated.
+
+=== Impacted Modules:
+
+==== Tendrl frontend impact:
+
+   * We need to add c3js chart (http://c3js.org/) libaray . and also one wrapper "angular patternfly" (http://angular-patternfly.rhcloud.com/) in order to access c3 chart features in angularjs application.
```
Comment (cybernth) [inline specs/donut_and_sparkline_chart_component.adoc:123]: Is there a dependency on the build/spec files?

Comment (kamleshverma1) [reply specs/donut_and_sparkline_chart_component.adoc:123]: No . not required .. automatically importing node_modules will take care for build/spec as well.

**Review by gnehapk — APPROVED** [review]

File: specs/donut_and_sparkline_chart_component.adoc (hunk 1 above)
Comment (cybernth) [reply specs/donut_and_sparkline_chart_component.adoc:123]: ok

**Review by cybernth — APPROVED** [review]

**Review by shtripat — COMMENTED** [review]
LGTM

**Pull request merged**